### PR TITLE
added CustomGlobalExceptionHandler class and validation for DTO class

### PR DESCRIPTION
**Pull request by AnyaRatusnyak**

### CODE REVIEW

**Review by slizko1 — APPROVED** [review]
GJ! I not sure do we need to do that but if we made GlobalEcxeptionHandler you can add handling of your EntityNotFoundException to it

File: src/main/java/projectbookstore/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -1,15 +1,22 @@
...
 import lombok.Getter;
 import lombok.Setter;
 
 @Getter
 @Setter
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
```
Comment (karma-o) [inline src/main/java/projectbookstore/dto/CreateBookRequestDto.java]: also let's add `@NotBlank` to ensure that we do not treat `""` or `"   "` like it is a valid data

File: src/main/java/projectbookstore/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -1,15 +1,22 @@
...
+    @NotNull
     private String isbn;
```
Comment (karma-o) [inline src/main/java/projectbookstore/dto/CreateBookRequestDto.java]: there is actually a specialized `@ISBN` for that purpose which we can use, and also I think that field is optional in our entity class, please double-check

File: src/main/java/projectbookstore/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -1,15 +1,22 @@
...
+    @NotNull
+    @Min(0)
     private BigDecimal price;
     private String description;
     private String coverImage;
```
Comment (karma-o) [inline src/main/java/projectbookstore/dto/CreateBookRequestDto.java]: I think we can also add `NotBlank` validation here

**Review by karma-o — APPROVED** [review]
Just some minor comments, GJ!

**Pull request merged**